### PR TITLE
update(docs): add descriptions of additional commands

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

**Review by o-andrieiev — DISMISSED** [review]

File: doc/additional_commands.rst [hunk 1]
```diff
@@ -67,7 +80,21 @@ Here are the parameters for poller mode:
 Detect changes and submit them automatically
 --------------------------------------------
 
-.. TODO: Add description
+Unlike default mode, Universum in `submit` mode **requires an already prepared local VCS.** For example:
```
Comment (o-andrieiev) [inline doc/additional_commands.rst:83]: prepared local VCS repo?

File: doc/additional_commands.rst [hunk 2]
```diff
@@ -51,7 +51,20 @@ Universum in non-CI mode has the following differences from default mode:
 Poll chosen VCS for updates
 ---------------------------
 
-.. TODO: Add description
+This mode was created for CI systems, not having polling feature at all or exact functionality, listed below.
```
Comment (i-keliukh) [inline doc/additional_commands.rst:54]: "This mode was created for CI systems that do not have polling feature at all or are not able to check each change individually."

File: doc/additional_commands.rst [hunk 3]
```diff
@@ -51,7 +51,20 @@ Universum in non-CI mode has the following differences from default mode:
...
+
+When launched in poller mode, Universum uses a simple database to remember for every passed VCS source
```
Comment (i-keliukh) [inline doc/additional_commands.rst:56]: "...to remember latest checked commit for every passed VCS source, such as Git or P4 branch."

File: doc/additional_commands.rst [hunk 4]
```diff
@@ -51,7 +51,20 @@ Universum in non-CI mode has the following differences from default mode:
...
+(such as Git or P4 branch) latest checked commit. On next launches it consults that database and calculates
+a list of updates (commits, submits) in that VCS source since last change saved in DB. After that, using an URL
+defined in parameters, it triggers a build in CI system for each of that changes (instead of testing only the
```
Comment (i-keliukh) [inline doc/additional_commands.rst:59]: that changes -> those changes

File: doc/additional_commands.rst [hunk 5]
```diff
@@ -67,7 +80,21 @@ Here are the parameters for poller mode:
...
+* In case of Git:
+
+    - the repo should be already cloned
+    - the required branch should be already checked out
+
+* In case of P4:
+
+    - the client should be already created
+    - the directory should be already synced
+    - all the required shelves should be applied
```
Comment (i-keliukh) [inline doc/additional_commands.rst:94]: Shelves? Did you mean changes?

Comment (k-dovgan) [reply doc/additional_commands.rst:94]: Well, in terms of Perforce I meant exactly shelves that should be unshelved if needed. _Before_ applying the actual "manual" changes.

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**